onnx/defs/parser.cc

```diff
@@ -43,6 +43,35 @@ Status OnnxParser::Parse(char open, IdList& idlist, char close) {
   return Status::OK();
 }
 
+Status OnnxParser::Parse(IdList& idlist, AttrList& attrlist) {
+  idlist.Clear();
+  attrlist.Clear();
+  do {
+    std::string id;
+    bool is_assignment = PeekIsIdentifierAssignment(id);
+    if (id.empty())
+      return Status::OK(); // Treat as empty list of identifiers
```

Comment:
minor nit: Not sure why we need this here. It has the effect of allowing "< >" (which is okay), as well as "< foo, >" (which is also okay, but we don't do that consistently everywhere else).